src/addrman.h

```diff
@@ -157,7 +157,7 @@ class CAddrInfo : public CAddress
 #define ADDRMAN_GETADDR_MAX_PCT 23
 
 //! the maximum number of nodes to return in a getaddr call
-#define ADDRMAN_GETADDR_MAX 2500
+#define ADDRMAN_GETADDR_MAX 1000
```

Comment:
Maybe add a static_assert in net.h that `MAX_ADDR_TO_SEND` is the same as `ADDRMAN_GETADDR_MAX` (with a TODO to remove `ADDRMAN_GETADDR_MAX`)